tests/gocase/util/math.go

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package util
+
+import "cmp"
+
+// Min returns the smaller of two values.
+func Min[T cmp.Ordered](a, b T) T {
```

Comment:
```suggestion
func Min[T cmp.Ordered](a, b T) T { //nolint:typecheck
```

Reply:
It should be a golangci-lint issue.

Reply:
@PragmaTwice Locally linter is OK with the code so maybe there is something in CI.